src/test/java/com/truelayer/java/payments/entities/schemeselection/userselected/SchemeSelectionTests.java

```diff
@@ -3,24 +3,11 @@
 import static org.junit.jupiter.api.Assertions.*;
 
 import com.truelayer.java.TrueLayerException;
-import java.util.stream.Stream;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.Arguments;
-import org.junit.jupiter.params.provider.MethodSource;
 
 public class SchemeSelectionTests {
 
-    @DisplayName("Scheme selection base class should yield the expected type and allow_remitter_fee")
```

Comment:
Removing this as the base class does not have the `allowRemitterFee` property anymore